### DISCUSSION TITLE
How to sync data directories with a remote PC

### DISCUSSION THREAD

**Discussion by Andrew-H2O** [felddy/foundryvtt-docker#642] [Q&A]
I'm looking for recommendations for how best to bi-directionally sync between the dockerized Foundry on my NAS and a copy of Foundry on a remote PC.

Could be a separate docker container, or even a foundry module (like the 'The Forge' module, for example).

Any suggestions?

**Comment by madereddy** [comment]
@Andrew-H2O I personally use syncthing containers to back up my NAS directory to my remote PC. It works seamlessly.
